File: Programmers/Inho/C#/Level01/자연수 뒤집어 배열로 만들기.cs
using System;
public class Solution {
    public int[] solution(long n) {
        int[] answer = new int[] {};
        char[] a = n.ToString().ToCharArray();
        System.Array.Reverse(a);
        answer = System.Array.ConvertAll(a, c => (int)System.Char.GetNumericValue(c));
        return answer;
    }
}

public class Solution {
    public int[] solution(long n) {
        string s = n.ToString();
        int[] answer = new int[s.Length];
        for(int i= 0; i<s.Length;i++){
            answer[i] = int.Parse(s[i].ToString());
        }
        Array.Reverse(answer);
        
        foreach(int i in answer){
            Console.Write(i);
        }
        return answer;
    }
}

File: Programmers/Inho/C++/Level01/자연수 뒤집어 배열로 만들기.cpp
#include <string>
#include <vector>
#include <iostream>
#include <algorithm>
using namespace std;

vector<int> solution(long long n) {
    vector<int> answer;
    string str = to_string(n);
    reverse(str.begin(), str.end());
    for(int i = 0; i< str.size();i++){
        answer.push_back(str[i]-'0');
    }
    
    return answer;
}